storage/spider/mysql-test/spider/t/create_temporary_table.test

```diff
@@ -0,0 +1,29 @@
+# This test tests for table creation option
+source include/have_log_bin.inc;
+--disable_warnings
+--disable_query_log
+--disable_result_log
+--source test_init.inc
+--source slave_test_init.inc
+--enable_result_log
+--enable_query_log
+
+--echo drop and create databases
+--connection master_1
+DROP DATABASE IF EXISTS auto_test_local;
+CREATE DATABASE auto_test_local;
+USE auto_test_local;
+
+--echo create temporary table
+DROP TABLE IF EXISTS tb_l;
+echo CREATE TEMPORARY TABLE tb_l (
+  a INT
+) ENGINE=Spider;
```

Comment:
Convention is to use the `MDEV-28225` in the `echo`. The echo on the `CREATE TABLE isn't necessary`. `DROP TABLE` is also unnecessary.

Like:
```
--echo #
--echo # MDEV-28225 Disallow user to create Spider temporary table
--echo #
```